### ISSUE TITLE
How i can start this project??

### ISSUE THREAD

**Issue by wonjunchoi-arc** [wty-ustc/HairCLIP#18]
sry , I'm begginer of ML , programming..
so I can't start this project with read me ...
plz help me

**Comment by wty-ustc** [comment]
You can try our HairCLIP on [Replicate](https://replicate.com/wty-ustc/hairclip), it does not require you to have ML background to experience it.

**Comment by wonjunchoi-arc** [comment]
Thank you for your answer! Actually, I've already done it with Replicate!
But I wanted to do it on my local, so I left this question!

**Comment by wty-ustc** [comment]
I'm not sure which step you're having trouble with, can you be more specific?

**Comment by wonjunchoi-arc** [comment]
um...
I did git clone and installed modules for virtual environment
And after that, I don't know how to train or test the model. The codes below are written as scripts/train.py, but I can't find the scripts folder in that git

And I can't find utils and models.psp among imported modules from predict.py

**Comment by wty-ustc** [comment]
Thanks for the suggestion, I have updated the readme, please check it out. Also, predict.py and cog.yaml are for Replicate only.

**Comment by wonjunchoi-arc** [comment]
Thank you. Can I follow the changed read me and If you don't mind me,  ask you again if I don't know anything?

**Comment by wonjunchoi-arc** [comment]
Can I use train_faces.pt as ref train data?

**Comment by wty-ustc** [comment]
No. You can get the dataset from this [repository](https://github.com/tkarras/progressive_growing_of_gans).

**Comment by ZziTaiLeo** [comment]
how can I resume training?

**Comment by xuzhi0413** [comment]
> Thank you. Can I follow the changed read me and If you don't mind me, ask you again if I don't know anything?



> Can I use train_faces.pt as ref train data?

Hi,I want to know where is train_faces.pt and test_faces.pt.I can not find them.
